src/cdomains/valueDomain.ml

```diff
@@ -223,12 +224,20 @@ struct
     | a, b -> a = b
 
   let cast_addr t a =
+    let rec patchTypeNotVLA t =
```

Comment:
Why do we need this here? Better name? `stripVarLenArr`, `onlyConstLengthArrays`...

Reply:
Good point, I'll change the name!

It is needed because `bitsSizeOf`, which is called a few lines down, can not handle types that contain arrays of non-const size as it is not possible to determine their size in bits from just to the type.